tests/crypto/psa_crypto/src/key_agreement.c

```diff
@@ -27,12 +28,11 @@ static const uint8_t server_public_key[] = {
 	0x8D, 0x49, 0x85, 0x8C, 0x7A, 0x9F, 0xC1, 0x46, 0xDA, 0xCC, 0x96,
 	0xEF, 0x6E, 0xD4, 0xDA, 0x71, 0xBF, 0xED, 0x32, 0x0D, 0x76,
 };
-static const uint8_t expected_shared_secret[] = {
+static const uint8_t expected_shared_secret[] __unused = {
```

Comment:
Usually, one of the benefits of `if (IS_ENABLED(CONFIG_XXXX))` is you don't need these annotations. Can you confirm you need this change?